concepts/05 Widgets/Diagram/10 Data Binding/40 Data Binding Options.md

```diff
@@ -0,0 +1,330 @@
+The **Diagram** widget allows you to bind [node's](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes) and [edge's](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/edges) settings to a database. Binding options have postfix _'Expr'_ and can contain a name of a data source field or an expression that returns the corresponding node's or edge's data.
```

Comment:
Change "node's" to "node";
Change "edge's" to "edge";

"Binding options have postfix _'Expr'_"
Can you be more specific than just "have"?

Change "and can contain a name of a data source field or an expression that returns the corresponding node's or edge's data" to "and can contain a data source field name or an expression that returns the corresponding node or edge data"